src/option.styles.ts

```diff
@@ -7,6 +7,7 @@ export default [
     }
 
     .container {
+      block-size: 1.75rem;
```

Comment:
Now 28 pixels tall instead of 27. One pixel difference. So I thought I'd skip a changeset for it. Eh?